package.json

```diff
@@ -15,16 +15,16 @@
     "reset-rn": "watchman watch-del-all; rm -rf $TMPDIR/metro-cache-*; rm -rf $TMPDIR/haste-map-*; rm -rf $TMPDIR/metro-symbolicate*",
     "reset-yarn": "yarn cache clean",
     "test": "yarn test-pure && yarn test-needing-ganache",
-    "test-pure": "yarn run lerna run test --ignore @celo/contractkit --ignore @celo/celocli --ignore @celo/transactions-uri",
+    "test-pure": "yarn workspaces foreach --all --exclude celo --exclude \"@celo/{celocli,contractkit,transactions-uri}\" -piv run test",
     "test-needing-ganache": "yarn workspace @celo/contractkit test && yarn workspace @celo/celocli test && yarn workspace @celo/transactions-uri test",
-    "build": "yarn run lerna run build",
-    "clean": "yarn run lerna run clean",
-    "docs": "yarn run lerna run docs",
+    "build": "yarn workspaces foreach --all -pitv run build",
```

Comment:
very pedantic but could we have the order of flags be the same for these three `-piv --all;`